### PR TITLE
A new skrellian plant, and associated stuff

### PR DESCRIPTION
**Pull request by VTCobaltblood**
Adds dyn, a Skrellian herb. Renowned as a medicine, its juice has mild anti-toxin properties ~~identical to how powerful tea was before Burger nerfed it;~~ lower than tea because Arrow considered old tea to be too OP, and any beverages made from it do too. Seeds are available both in the garden and hydroponics, and the plant may occasionally contain actual dylovene in it. 
The new drinks are the raw juice of the herb, a tea (1 juice to 4 water), and an icy/fizzy iced tea _thing_ (1 juice to 2 ice to 2 soda water). The last comes as a soda can in the vendor, and a carton of juice is available at the bartender's boozemat.
This is the first in a series of PRs expanding upon skrellian cuisine/drinks.

### CODE REVIEW

**Comment by VTCobaltblood** [comment]
!review

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm [hunk 1]
```diff
@@ -933,6 +933,45 @@
...
+	glass_name = "glass of dyn juice"
+	glass_desc = "Juice from a dyn leaf. Good for you, but normally not consumed undiluted."
+
+/datum/reagent/drink/dynjuice/affect_ingest(var/mob/living/carbon/M, var/alien, var/removed)
+	..()
+	M.adjustToxLoss(-0.5 * removed)
+
+
+/datum/reagent/drink/dynjuice/affect_blood(var/mob/living/carbon/M, var/alien, var/removed)
+		M.adjustToxLoss(-0.5 * removed)
```
Comment (Arrow768) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm]:
```suggestion
	M.adjustToxLoss(-0.5 * removed)
```

File: code/game/machinery/vending_types.dm [hunk 2]
```diff
@@ -206,11 +207,12 @@
 		/obj/item/weapon/reagent_containers/food/drinks/cans/root_beer = 10,
 		/obj/item/weapon/reagent_containers/food/drinks/cans/starkist = 10,
 		/obj/item/weapon/reagent_containers/food/drinks/cans/waterbottle = 10,
+		/obj/item/weapon/reagent_containers/food/drinks/cans/dyn = 10,
 		/obj/item/weapon/reagent_containers/food/drinks/cans/space_up = 10,
 		/obj/item/weapon/reagent_containers/food/drinks/cans/iced_tea = 10,
 		/obj/item/weapon/reagent_containers/food/drinks/cans/grape_juice = 10,
 		/obj/item/weapon/reagent_containers/food/drinks/cans/koispunch = 5,
-		/obj/item/weapon/reagent_containers/food/drinks/cans/beetle_milk = 10
+		/obj/item/weapon/reagent_containers/food/drinks/cans/beetle_milk = 10,
```
Comment (Arrow768) [inline code/game/machinery/vending_types.dm]:
```suggestion
		/obj/item/weapon/reagent_containers/food/drinks/cans/beetle_milk = 10
```

**Comment by VTCobaltblood** [comment]
The tea used to heal the same damage and also was available in vendors, with the same volume. And it was removed for HRP reasons and not balance reasons. I don't see a problem with it.

**Comment by ParadoxSpace** [comment]
Toxin damage barely gets fixed on its' own by the body. It's annoying to have 0.1 toxin damage that will never be fixed.

**Comment by Conspiir** [comment]
My only hang-up is the dyn seed is not alphabetized in the seed vendor lists.

Otherwise, I think it looks great!

**Comment by TehFlaminTaco** [comment]
> 
> 
> Toxin damage barely gets fixed on its' own by the body. It's annoying to have 0.1 toxin damage that will never be fixed.

Toxins can be fixed with *vomit

**Pull request merged**